spec/api-deprecations-spec.js

```diff
@@ -89,7 +89,7 @@ describe('deprecations', () => {
     deprecations.setHandler(m => { msg = m })
 
     const propertyName = 'itMustGo'
-    const o = { [propertyName]: 0 }
+    let o = { [propertyName]: 0 }
```

Comment:
The contents of `o` change, but I don't think `o` points to a different object after `deprecate.removeProperty` -- can this stay `const`?

Reply:
oh yes oops, i was trying to suss out failures last night kinda late heh